src/patternfly/components/Page/page.scss

```diff
@@ -37,31 +37,12 @@ $pf-c-page--breakpoint-map: build-breakpoint-map("base", "sm", "md", "lg", "xl",
 
   // Header nav
   --pf-c-page__header-nav--BackgroundColor: var(--pf-global--BackgroundColor--dark-300);
-  --pf-c-page__header-nav--lg--BackgroundColor: transparent;
-  --pf-c-page__header-nav--PaddingRight: 0;
-  --pf-c-page__header-nav--PaddingLeft: 0;
-  --pf-c-page__header-nav--md--PaddingRight: var(--pf-global--spacer--sm);
-  --pf-c-page__header-nav--md--PaddingLeft: var(--pf-global--spacer--sm);
-  --pf-c-page__header-nav--xl--PaddingRight: 0;
-  --pf-c-page__header-nav--xl--PaddingLeft: 0;
-  --pf-c-page__header--c-nav__scroll-button--disabled--Color: var(--pf-global--disabled-color--100);
-  --pf-c-page__header--c-nav__scroll-button--before--BorderColor: var(--pf-global--BackgroundColor--dark-200);
-
-  // --pf-c-page__header--c-nav__scroll-button--disabled--before--BorderColor: var(--pf-global--BackgroundColor--dark-200);
-  --pf-c-page__header--c-nav__scroll-button--disabled--before--BorderColor: transparent;
-
-  @media screen and (min-width: $pf-global--breakpoint--xl) {
-    --pf-c-page__header-nav--PaddingLeft: var(--pf-c-page__header-nav--xl--PaddingLeft);
-    --pf-c-page__header-nav--c-nav__scroll-button--nth-of-type-1--Left: var(--pf-c-page__header-nav--c-nav__scroll-button--nth-of-type-1--xl--Left);
-  }
-
-  @media screen and (min-width: $pf-global--breakpoint--lg) {
```

Comment:
did you mean to change the breakpoint where the nav moves to the bottom of the masthead?